src/components/Header/Header.jsx

```diff
@@ -88,9 +88,12 @@ const LinkGroup = ({ navbarPages }) => {
     setAnchorEl(undefined);
   }, [anchorEl]);
 
+  const mainNavbarPages = useMemo(() => navbarPages.slice(0, 3), [navbarPages]);
```

Comment:
Rather than slicing by index, I think we should add a property in each navbar object that indicates whether it's a main one or a drawer one, and render the drawer ones in a burger menu as described in https://github.com/odota/web/issues/2222